### DISCUSSION TITLE
Welcome to mobly Discussions!

### DISCUSSION THREAD

**Discussion by xpconanfan** [google/mobly#829] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anurag01a** [comment]
Namaste Everyone🙏! I am Anurag, a CS Junior from India scratching my way into tech! 
What's Up Everybody Howz It Going!!😄

**Comment by B2BM2M** [comment]
Hi all,
I downloaded the mobly bundle snippets on Linux.
When I ran the ./gradlew assembleDebug, I got this error saying "SDK location not found. Define location with an ANDROID_SDK_ROOT environment variable".

I took the initial step  to add the SDK path in the .bashrc file but, I'm still having the same issue.

Any suggestion?
